templates/zerver/help/delete-a-message.md

```diff
@@ -0,0 +1,7 @@
+# Delete a message
+
+{!admin-only.md!}
```

Comment:
I think the plan is to change this feature to not be admin-only, so it may be OK for the macro to be slightly awkward?  Idunno.